jitbuilder/compile/Method.hpp

```diff
@@ -166,7 +167,7 @@ class ResolvedMethod : public ResolvedMethodBase, public Method
                                TR::SymbolReferenceTable *symRefTab);
 
    protected:
-   const char *_fileName;
+   std::string _fileName;
```

Comment:
I am pretty confident, the underlying character array is on the heap (or global data segment?). It's the responsibility of the `std::string` class to manage that, and therefore, it will get deallocated automatically once the corresponding `std::string` is no longer being used.

Just curious, any particular reason why you are interested about this?